api/internal/core/store/store.go

```diff
@@ -267,6 +274,9 @@ func (s *GenericStore) Update(ctx context.Context, obj interface{}) error {
 	}
 	oldObj, ok := s.cache.Load(key)
 	if !ok {
+		if createOnFail {
+			return s.Create(ctx, obj)
```

Comment:
We should init `create_time`、`update_time`  and `id`，Hmmm，we can add two function for baseinfo: `creating` and `updating` to do some logic about `BaseInfo`, then add `BaseInfoSetter` interface to use it.